hub/policy/src/test/java/uk/gov/ida/hub/policy/domain/state/serialization/DeserializationTest.java

```diff
@@ -102,10 +102,10 @@ public void teardown() {
     }
 
     @Test
-    public void shouldDeserializeSessionStartedState() throws Exception {
```

Comment:
Why can this be removed?

Reply:
Because this method would never throw a checked exception. Java has an exception propagation mechanism that defines this. It's usually not obvious when a method may or may not throw an unchecked exception - I rely on the IDE (IntelliJ) to tell me